resqpy/fault/_grid_connection_set.py

```diff
@@ -1236,8 +1236,21 @@ def create_xml(self,
 
         return gcs
 
-    def write_simulator(self, filename, mode = 'w', simulator = 'nexus', include_both_sides = False, use_minus = False):
-        """Creates a Nexus include file holding MULT keywords and data."""
+    def write_simulator(self,
+                        filename,
+                        mode = 'w',
+                        simulator = 'nexus',
+                        include_both_sides = False,
+                        use_minus = False,
+                        trans_mult_uuid = None):
+        """Creates a Nexus include file holding MULT keywords and data. trans_mult_uuid (optional) is the uuid of a property on the gcs containing transmissibility multiplier values. If not provided values of 1.0 will be used."""
+        if trans_mult_uuid is not None:
+            assert self.property_collection.part_in_collection(self.model.part_for_uuid(
```

Comment:
Best to call the self.extract_property_collection() method before this assertion, to ensure that the property_collection is populated.